### PR TITLE
Refactored the creation screen and mostly added comments

### LINKED ISSUE #320: Refactor creation screen - Title and Description

### PR DESCRIPTION
**Pull request by imaneb942**
**Closes #320 **
- changed a few colors scheme in the creation screen process
- reduced the size of the window in the map part of the creation process
- added an indication title to the Title and Description part of the process
- commented my previously done functions

### CODE REVIEW

File: app/src/main/java/com/github/wanderwise_inc/app/ui/creation/steps/CreateItineraryMap.kt
Comment (engjellismaili) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/creation/steps/CreateItineraryMap.kt:1]: Clean comments

File: app/src/main/java/com/github/wanderwise_inc/app/ui/creation/steps/CreationStepChooseDescription.kt [hunk 1]
```diff
@@ -2,10 +2,17 @@ package com.github.wanderwise_inc.app.ui.creation.steps
...
+import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
 import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Divider
```
Comment (engjellismaili) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/creation/steps/CreationStepChooseDescription.kt:15]: is the Divider only avaible in material, or is there an equivalent Divider in material3 ?

File: app/src/main/java/com/github/wanderwise_inc/app/ui/creation/steps/CreationStepChooseDescription.kt
Comment (engjellismaili) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/creation/steps/CreationStepChooseDescription.kt:1]: Lgtm well done

File: app/src/main/java/com/github/wanderwise_inc/app/ui/home/SearchBar.kt
Comment (engjellismaili) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/home/SearchBar.kt:1]: Good comments

**Review by engjellismaili — APPROVED** [review]
Lgtm, clean comments

File: app/src/main/java/com/github/wanderwise_inc/app/ui/creation/steps/CreationStepChooseDescription.kt (hunk 1 above)
Comment (imaneb942) [reply app/src/main/java/com/github/wanderwise_inc/app/ui/creation/steps/CreationStepChooseDescription.kt:15]: probably, I will change it

File: app/src/main/java/com/github/wanderwise_inc/app/ui/creation/steps/CreationStepChooseDescription.kt [hunk 2]
```diff
@@ -32,72 +41,92 @@ fun CreationStepChooseDescriptionScreen(createItineraryViewModel: CreateItinerar
...
-        }
-
-        item {
-          if (!validTitle) {
-            Text(
-                text = createItineraryViewModel.invalidTitleMessage(),
-                color = Color.Red,
-                modifier = Modifier.testTag(TestTags.INVALID_INPUT))
+                    text = createItineraryViewModel.invalidTitleMessage(),
+                    color = Color.Red,
```
Comment (276746) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/creation/steps/CreationStepChooseDescription.kt:101]: Great addition to show the error

**Review by 276746 — APPROVED** [review]
Good comment and refreshing theme changes, nice work

**Pull request merged**